workspaces/blackduck/plugins/blackduck-backend/report.api.md

```diff
@@ -4,32 +4,14 @@
 
 ```ts
 import { BackendFeature } from '@backstage/backend-plugin-api';
+import { BlackDuckConfig } from '@backstage-community/plugin-blackduck-node';
 import { Config } from '@backstage/config';
 import { DiscoveryService } from '@backstage/backend-plugin-api';
 import express from 'express';
 import { HttpAuthService } from '@backstage/backend-plugin-api';
 import { LoggerService } from '@backstage/backend-plugin-api';
 import { PermissionsService } from '@backstage/backend-plugin-api';
 
-// @public
-export class BlackDuckConfig {
-  constructor(hosts: BlackDuckHostConfig[], defaultHost: string);
-  // (undocumented)
-  static fromConfig(config: Config): BlackDuckConfig;
-  // (undocumented)
-  getHostConfigByName(name: string): BlackDuckHostConfig;
-}
-
-// @public (undocumented)
-export interface BlackDuckHostConfig {
-  // (undocumented)
-  host: string;
-  // (undocumented)
-  name: string;
-  // (undocumented)
-  token: string;
-}
```

Comment:
this should be marked as breaking

Reply:
This isn't really a breaking change, `BlackDuckConfig & BlackDuckHostConfig` are moved to `plugin-blackduck-node` and no changes are required on `plugin-blackduck-backend` for the integrators post plugin upgrade.

Reply:
yeah but since these were exported by the plugin, adopters could have imported the types which means that their code will break